src/api/goods.ts

```
@@ -4,13 +4,21 @@ import { Good } from '../types/Good';
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
 export function getAll(): Promise<Good[]> {
-  return fetch(API_URL).then(response => response.json());
+  return fetch(API_URL).then(response => {
```

The getAll implementation throws when `response.ok` is false which prevents silently returning bad data. The checklist requires handling possible API errors: "3. [API] - When you make a request to the server, don't forget to handle possible errors" . 

This implementation is acceptable, but make sure callers of these methods (e.g., the App component) handle promise rejections (with `.catch(...)` or `try/catch` around `await`) and surface a user-visible error state. Alternatively, you may choose to normalize errors here (catch and rethrow a consistent Error) so callers receive predictable error messages.